# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# marcin mikołajczak <me@mkljczk.pl>, 2017
# KagiSame, 2018
# Piotr Strębski <strebski@gmail.com>, 2020
# Jacob B. <brickminerplyt@gmail.com>, 2021
# cooky, 2023
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-09-28 22:49+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: cooky, 2023\n"
"Language-Team: Polish (https://app.transifex.com/calamares/teams/20061/pl/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: pl\n"
"Plural-Forms: nplurals=4; plural=(n==1 ? 0 : (n%10>=2 && n%10<=4) && (n%100<12 || n%100>14) ? 1 : n!=1 && (n%10>=0 && n%10<=1) || (n%10>=5 && n%10<=9) || (n%100>=12 && n%100<=14) ? 2 : 3);\n"

#: src/modules/bootloader/main.py:46
msgid "Install bootloader."
msgstr "Instalacja programu rozruchowego."

#: src/modules/bootloader/main.py:644
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""
"Nie udało się zainstalować GRUBa, nie zdefiniowano partycji w globalnej "
"pamięci masowej"

#: src/modules/bootloader/main.py:899
msgid "Bootloader installation error"
msgstr "Błąd instalacji bootloadera"

#: src/modules/bootloader/main.py:900
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Nie można zainstalować bootloadera. Polecenie instalacyjne <pre>{!s}</pre> "
"zwróciło kod błędu {!s}."

#: src/modules/displaymanager/main.py:509
msgid "Cannot write LXDM configuration file"
msgstr "Nie można zapisać pliku konfiguracji LXDM"

#: src/modules/displaymanager/main.py:510
msgid "LXDM config file {!s} does not exist"
msgstr "Plik konfiguracji LXDM {!s} nie istnieje"

#: src/modules/displaymanager/main.py:598
msgid "Cannot write LightDM configuration file"
msgstr "Nie można zapisać pliku konfiguracji LightDM"

#: src/modules/displaymanager/main.py:599
msgid "LightDM config file {!s} does not exist"
msgstr "Plik konfiguracji LightDM {!s} nie istnieje"

#: src/modules/displaymanager/main.py:684
msgid "Cannot configure LightDM"
msgstr "Nie można skonfigurować LightDM"

#: src/modules/displaymanager/main.py:685
msgid "No LightDM greeter installed."
msgstr "Nie zainstalowano ekranu powitalnego LightDM."

#: src/modules/displaymanager/main.py:716
msgid "Cannot write SLIM configuration file"
msgstr "Nie można zapisać pliku konfiguracji SLIM"

#: src/modules/displaymanager/main.py:717
msgid "SLIM config file {!s} does not exist"
msgstr "Plik konfiguracji SLIM {!s} nie istnieje"

#: src/modules/displaymanager/main.py:935
msgid "No display managers selected for the displaymanager module."
msgstr "Brak wybranych menedżerów wyświetlania dla modułu displaymanager."

#: src/modules/displaymanager/main.py:936
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""
"Lista displaymanagers jest pusta lub niezdefiniowana w globalstorage oraz "
"displaymanager.conf."

#: src/modules/displaymanager/main.py:1023
msgid "Display manager configuration was incomplete"
msgstr "Konfiguracja menedżera wyświetlania była niekompletna"

#: src/modules/dracut/main.py:29
msgid "Creating initramfs with dracut."
msgstr "Tworzenie initramfs z dracut."

#: src/modules/dracut/main.py:63
msgid "Failed to run dracut"
msgstr "Nie udało się uruchomić dracut"

#: src/modules/dracut/main.py:64
#, python-brace-format
msgid "Dracut failed to run on the target with return code: {return_code}"
msgstr ""
"Nie udało się uruchomić programu Dracut na celu z kodem błędu: {return_code}"

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Zadanie fikcyjne Python."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Krok fikcyjny Python {}"

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr "Zapisywanie fstab."

#: src/modules/fstab/main.py:377 src/modules/fstab/main.py:383
#: src/modules/fstab/main.py:411 src/modules/initcpiocfg/main.py:256
#: src/modules/initcpiocfg/main.py:260 src/modules/initramfscfg/main.py:85
#: src/modules/initramfscfg/main.py:89 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:329 src/modules/networkcfg/main.py:105
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr "Błąd konfiguracji"

#: src/modules/fstab/main.py:378 src/modules/initramfscfg/main.py:86
#: src/modules/mount/main.py:330 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "Nie ma zdefiniowanych partycji dla <pre>{!s}</pre> do użytku."

#: src/modules/fstab/main.py:384 src/modules/initramfscfg/main.py:90
#: src/modules/localecfg/main.py:141 src/modules/networkcfg/main.py:106
#: src/modules/openrcdmcryptcfg/main.py:77
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""
"Nie znaleziono głównego punktu montowania dla <pre>{!s}</pre> do użycia."

#: src/modules/fstab/main.py:412
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr "Nie podano konfiguracji <pre>{!s}</pre> dla <pre>{!s}</pre>."

#: src/modules/grubcfg/main.py:29
msgid "Configure GRUB."
msgstr "Konfiguracja GRUB."

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "Ustawianie zegara systemowego."

#: src/modules/initcpiocfg/main.py:27
msgid "Configuring mkinitcpio."
msgstr "Konfigurowanie mkinitcpio."

#: src/modules/initcpiocfg/main.py:257
msgid "No partitions are defined for <pre>initcpiocfg</pre>."
msgstr "Nie ma zdefiniowanych partycji dla <pre>initcpiocfg</pre>."

#: src/modules/initcpiocfg/main.py:261
msgid "No root mount point for <pre>initcpiocfg</pre>."
msgstr "Brak głównego punktu montowania dla <pre>initcpiocfg</pre>."

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "Konfigurowanie initramfs."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Konfigurowanie ustawień lokalnych."

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "Tworzenie initramfs z mkinitfs."

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "Nie udało się włączyć mkinitfs."

#: src/modules/mkinitfs/main.py:50
msgid "The exit code was {}"
msgstr "Kod wyjściowy to {}"

#: src/modules/mount/main.py:43
msgid "Mounting partitions."
msgstr "Montowanie partycji."

#: src/modules/mount/main.py:164 src/modules/mount/main.py:200
msgid "Internal error mounting zfs datasets"
msgstr "Wewnętrzny błąd montowania zestawów danych zfs"

#: src/modules/mount/main.py:176
msgid "Failed to import zpool"
msgstr "Nie udało się zaimportować zpool"

#: src/modules/mount/main.py:192
msgid "Failed to unlock zpool"
msgstr "Nie udało się odblokować zpool"

#: src/modules/mount/main.py:209 src/modules/mount/main.py:214
msgid "Failed to set zfs mountpoint"
msgstr "Nie udało się ustawić punktu montowania zfs"

#: src/modules/mount/main.py:365
msgid "zfs mounting error"
msgstr "Błąd montowania zfs"

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr "Zapisywanie konfiguracji sieci."

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "Konfigurowanie usługi OpenRC dmcrypt."

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Zainstaluj pakiety."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "Przetwarzanie pakietów (%(count)d / %(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "Instalowanie jednego pakietu."
msgstr[1] "Instalowanie %(num)d pakietów."
msgstr[2] "Instalowanie %(num)d pakietów."
msgstr[3] "Instalowanie%(num)d pakietów."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Usuwanie jednego pakietu."
msgstr[1] "Usuwanie %(num)d pakietów."
msgstr[2] "Usuwanie %(num)d pakietów."
msgstr[3] "Usuwanie %(num)d pakietów."

#: src/modules/packages/main.py:740 src/modules/packages/main.py:752
#: src/modules/packages/main.py:780
msgid "Package Manager error"
msgstr "Błąd Menedżera pakietów"

#: src/modules/packages/main.py:741
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""
"Menedżer pakietów nie może przygotować aktualizacji. Polecenie "
"<pre>{!s}</pre> zwróciło kod błędu {!s}."

#: src/modules/packages/main.py:753
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""
"Menedżer pakietów nie może zaktualizować systemu. Polecenie <pre>{!s}</pre> "
"zwróciło kod błędu {!s}."

#: src/modules/packages/main.py:781
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Menedżer pakietów nie może wprowadzać zmian w zainstalowanym systemie. "
"Polecenie <pre>{!s}</pre> zwróciło kod błędu {!s}."

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Konfiguracja motywu Plymouth"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "Instalowanie danych."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "Konfiguracja usług OpenRC"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr ""
"Nie udało się dodać usługi {name!s} do poziomu-uruchamiania {level!s}."

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr ""
"Nie udało się usunąć usługi {name!s} do poziomu-uruchamiania {level!s}."

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"Nieznana akcja-usługi <code>{arg!s}</code> dla usługi {name!s} w poziomie-"
"uruchamiania {level!s}."

#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "Nie można zmodyfikować usług"

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"<code>rc-update {arg!s}</code> wezwanie w chroot zwróciło kod błędu {num!s}."

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "Docelowy poziom odtwarzania nie istnieje"

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"Ścieżka do poziomu odtwarzania {level!s} to <code>{path!s}</code>,  nie "
"istnieje."

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "Docelowa usługa nie istnieje"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr "Ścieżka do usługi {name!s} to <code>{path!s}</code>,  nie istnieje."

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd units"
msgstr "Konfiguracja jednostek systemd"

#: src/modules/services-systemd/main.py:64
msgid "Cannot modify unit"
msgstr "Nie można zmodyfikować jednostki"

#: src/modules/services-systemd/main.py:65
msgid ""
"<code>systemctl {_action!s}</code> call in chroot returned error code "
"{_exit_code!s}."
msgstr ""
"<code>systemctl {_action!s}</code> wywołanie chroot zwróciło kod błędu "
"{_exit_code!s}."

#: src/modules/services-systemd/main.py:66
msgid "Cannot {_action!s} systemd unit <code>{_name!s}</code>."
msgstr "Nie można  {_action!s} jednostki systemd <code>{_name!s}</code>."

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "Zapełnianie systemu plików."

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr "rsync zakończyło działanie kodem błędu {}."

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "Odpakowywanie obrazu {}/{}, pliku {}/{}"

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr "Rozpoczynanie odpakowywania {}"

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr "Błąd rozpakowywania obrazu \"{}\""

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr "Brak punktu montowania partycji root"

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr "globalstorage nie zawiera klucza \"rootMountPoint\"(punktu montowania /)."

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr "Błędny punkt montowania partycji root"

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr " Punkt montowania / to \"{}\", który nie istnieje."

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr "Niewłaściwa konfiguracja unpackfs"

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr "Nie ma żadnych informacji konfiguracyjnych."

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr ""
"System plików dla \"{}\" ({}) nie jest wspierany przez obecne jądro twojego "
"systemu"

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr "Źródłowy system plików \"{}\" nie istnieje"

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""
"Nie udało się znaleźć unsquashfs, upewnij się, że masz zainstalowany pakiet "
"squashfs-tools."

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "Miejsce docelowe \"{}\" w docelowym systemie nie jest katalogiem"

#: src/modules/zfshostid/main.py:27
msgid "Copying zfs generated hostid."
msgstr "Kopiowanie hostid wygenerowanego przez zfs."
